plugins/org.python.pydev.debug/src_console/org/python/pydev/debug/newconsole/PydevConsoleConstants.java

```diff
@@ -85,6 +89,29 @@
             { "OpenGL (glut)", "glut" },
             { "pyglet (pyglet)", "pyglet" } };
 
+    public static final String ACTIVE_EDITOR_INTERPRETER_REPRESENTATION = "active editor";
+    public static final String PYTHON_INTERPRETER_REPRESENTATION = "python";
+    public static final String PYDEV_INTERPRETER_REPRESENTATION = "pydev";
+    public static final String JYTHON_INTERPRETER_REPRESENTATION = "jython";
+    public static final String IRONPYTHON_INTERPRETER_REPRESENTATION = "ironpython";
+    public static final String INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER = "INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER";
+    public static final String DEFAULT_INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER = "none";
+    public static final String[][] ENTRIES_VALUES_INTERACTIVE_CONSOLE_DEFAULT_INTERPRETER = new String[][] {
+            { "Always ask for console type", "none" },
+            { "Console for currently active editor", ACTIVE_EDITOR_INTERPRETER_REPRESENTATION },
+            { "Python Console", PYTHON_INTERPRETER_REPRESENTATION },
+            { "Pydev Debug Console", PYDEV_INTERPRETER_REPRESENTATION },
+            { "Jython Console", JYTHON_INTERPRETER_REPRESENTATION },
+            { "IronPython Console", IRONPYTHON_INTERPRETER_REPRESENTATION }
+    };
+    public static final Set<String> INTERACTIVE_CONSOLE_INTERPRETERS_LIST = new HashSet<String>(Arrays.asList(
```

Comment:
Since this is a `Set` and not a `List` I think you could remove the postfix i.e.: Rename to: `INTERACTIVE_CONSOLE_INTERPRETERS`.